client/app/assets/less/redash/query.less

```diff
@@ -538,6 +538,16 @@ nav .rg-bottom {
   padding: 0 0 0 23px;
 }
 
+.table--permission {
+  .profile__image {
+    margin-right: 0;
+  }
+}
+
+.mp__permission-type {
```

Comment:
What's `mp`? And why two styles of naming? (`--` vs `__`)

Reply:
mp refers to manage permission but ended up not using elsewhere, maybe the table should be mp__table but I chose to keep it aligned with the other table class names: "table table-condensed table-hover table--permission".

That's not 2 styles, it's BEM: http://getbem.com/naming/